vite.config.js

```diff
@@ -18,6 +18,14 @@ export default defineConfig({
   build: {
     sourcemap: true,
     emptyOutDir: true,
+    rollupOptions: {
+      onwarn: (warning, defaultHandler) => {
+        if (warning.code === "INVALID_ANNOTATION" && warning.message.includes("*#__PURE__*")) {
```

Comment:
Let's create an issue in SForms, referencing this commit and other related links.

Reply:
Will be resolved by https://github.com/kbss-cvut/s-forms/issues/282.